s3_start_transfer/src/archivematica.py

```diff
@@ -0,0 +1,181 @@
+# -*- encoding: utf-8
+"""
+Code for talking to Archivematica.
+"""
+
+import base64
+import collections
+import os
+
+import requests
+
+
+class StartTransferException(Exception):
+    pass
+
+
+class StoragePathException(Exception):
+    pass
+
+
+def am_api_post_json(url, data):
+    """
+    POST json to the Archivematica API
+    :param api_path: URL path to request (without hostname, e.g. /api/v2/location/)
+    :param data: Dict of data to post
+    :returns: dict of json data returned by request
+    """
+    am_url = os.environ["ARCHIVEMATICA_URL"]
+    am_user = os.environ["ARCHIVEMATICA_USERNAME"]
+    am_api_key = os.environ["ARCHIVEMATICA_API_KEY"]
+    am_headers = {"Authorization": "ApiKey {0}:{1}".format(am_user, am_api_key)}
```

Comment:
f strings?

Reply:
Or are we in py2 land?

Reply:
This is code that came from Py2 land, will fix.